### ISSUE TITLE
Can i update demo of angular2+ in readme

### ISSUE THREAD

**Issue by nasreddineskandrani** [redux-observable/redux-observable#365]
Hi,
I want to add an example for angular2 but not using JSBin. Are you okay with that?
Also their is no example for  lazy loading support (replaceEpic usage). Do you want us to provide one for angular2+ and React? or maybe their is and i didn't find it?

**Comment by jayphelps** [comment]
All of those sound amazing! Thank you! 🕺

**Comment by nasreddineskandrani** [comment]
what do you think about [stackblitz](https://stackblitz.com/), to host the demos? do you have any other website to suggest before we start?

**Comment by nasreddineskandrani** [comment]
simple example PR: https://github.com/redux-observable/redux-observable/pull/366/commits/f5a9d918a949b6c33195d535fcfb0978a785589a

Lazy loading  example for angular coming soon.

**Comment by jayphelps** [comment]
Tanks again!

**Comment by nasreddineskandrani** [comment]
No Problem. You did well closing this issue since the main target was to add a live example ng2+ on ReadMe.

Still i want to do something about lazyloading examples. My friend already start something with React.
https://stackblitz.com/edit/react-redux-observable-async?file=Hello.js
But it's for the moment a quick first draft prototype. I didn't have time to review/test it yet (big time at the job these dayz).

So when we have a final example for lazy loading React or Angular Epics :) i don't know which one will be first. i ll create a new issue for it.

**Comment by nasreddineskandrani** [comment]
voila first draft working-> lazy loaded epic in React with typescript.
https://github.com/nasreddineskandrani/example-react-lazyload-epics
If someone has a comment don't hesitate. 

I didn't think about it much but to unload the lazyloaded epics i had to handle that with an `unload action`. I need to think more to see if it need some code change in the lib to handle it without this action unload dependency.
